packages/app/src/runner/aut-iframe.ts

```diff
@@ -307,22 +311,22 @@ export class AutIframe {
       // bail if our el no longer exists in the parent body
       if (!this.$.contains(body, element)) return
 
-      // switch to using outerWidth + outerHeight
+      // switch to using offsetWidth + offsetHeight
       // because we want to highlight our element even
       // if it only has margin and zero content height / width
-      const dimensions = this.dom.getOuterSize($_el)
+      const dimensions = this._getOffsetSize($_el.get(0))
```

Comment:
jquery `outerWidth` and `outerHeight` are equal to `offsetWidth` and `offsetHeight` when [only first argument is given to the function](https://github.com/cypress-io/cypress/blob/566a1a208cb51d193fd1838a4c35ab872a729fe7/packages/runner/src/dom/dimensions.js#L79-L84).

Compare the images.

* JQuery `outerWidth`

![](https://api.jquery.com/resources/0042_04_06.png)

[doc link](https://api.jquery.com/outerwidth/)

* Vailla js `offsetWidth`

![](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetWidth/dimensions-offset.png)

[doc link](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetWidth)